app/webpacker/components/Panel/pages/FixResultsPage/api/getEvents.js

```diff
@@ -0,0 +1,12 @@
+import { fetchJsonOrError } from '../../../../../lib/requests/fetchWithAuthenticityToken';
+import { viewUrls } from '../../../../../lib/requests/routes.js.erb';
+
+async function getEvents({ wcaId, competitionId }) {
+  const { data } = await fetchJsonOrError(
+    viewUrls.persons.results(wcaId, competitionId),
+  );
```

Comment:
You can actually use the `getResults` API function (that you're adding in this PR as well) for this